components/reusable-components/login-form/LoginForm.module.scss

```diff
@@ -149,3 +155,16 @@
 .checkboxDark {
   border: 1.5px solid var(--Secondary, hsla(220, 20%, 43%, 1));
 }
+.errorMessageContainer {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  background-color: #fff1f1; /* Light red background for emphasis */
+  border: 1px solid #ff6b6b; /* Slightly darker border */
```

Comment:
we have predifined colors and styles you can see them in _colors.scss and typography.scss, please incorporate  those styles in the new classes you have added here